### PR TITLE
Fixes links to social urls

### PR DESCRIPTION
**Pull request by matt-bernhardt**
This updates the links to our social media profiles.

Ticket: https://mitlibraries.atlassian.net/browse/ENGX-60

### CODE REVIEW

File: _includes/footer-site-slim.html [hunk 1]
```diff
@@ -15,27 +15,27 @@
...
                 <div class="wrap-social">
                     <p class="text-find-us">Find us on</p>
-                    <a href="//twitter.com/mitlibraries" title="Twitter">
+                    <a href="https://twitter.com/mitlibraries" title="Twitter">
                         <svg class="icon-social--twitter" width="2048" height="2048" viewBox="-192 -384 2048 2048" xmlns="http://www.w3.org/2000/svg"><g transform="scale(1 -1) translate(0 -1280)"><path d="M1620 1128q-67 -98 -162 -167q1 -14 1 -42q0 -130 -38 -259.5t-115.5 -248.5t-184.5 -210.5t-258 -146t-323 -54.5q-271 0 -496 145q35 -4 78 -4q225 0 401 138q-105 2 -188 64.5t-114 159.5q33 -5 61 -5q43 0 85 11q-112 23 -185.5 111.5t-73.5 205.5v4q68 -38 146 -41 q-66 44 -105 115t-39 154q0 88 44 163q121 -149 294.5 -238.5t371.5 -99.5q-8 38 -8 74q0 134 94.5 228.5t228.5 94.5q140 0 236 -102q109 21 205 78q-37 -115 -142 -178q93 10 186 50z" fill="black"></path></g></svg>
                         <span class="sr">Twitter</span>
                     </a><!-- End Twitter -->
                     
-                    <a href="/facebook" title="Facebook">
+                    <a href="https://libraries.mit.edu/facebook" title="Facebook">
```
Comment (JPrevost) [inline _includes/footer-site-slim.html:23]: Is there a reason we use an mit.edu link for Facebook and not `https://www.facebook.com/mitlib`? This works fine but I'm curious about the inconsistency with the different links.

**Review by JPrevost — APPROVED** [review]
I'm 👍🏻 if we have a solid reason to use a redirect to Facebook and not the other sites. If it's just legacy I'd prefer to change to a direct link now before we roll this out everywhere. If we have any reason to leave it as is though, :shipit:

File: _includes/footer-site-slim.html (hunk 1 above)
Comment (JPrevost) [reply _includes/footer-site-slim.html:23]: Let's move this to a direct link to match the rest.

**Comment by matt-bernhardt** [comment]
I've updated the Facebook link after further discussions with UX and within our team... merging now.

**Pull request merged**